train-mssg.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/sh
-
-if [ ! -e text8 ]; then
-  wget https://dl.dropboxusercontent.com/u/39534006/text8.zip
```

Comment:
wrangled corpus derived from Wikipedia English often used for text compression or processing benchmarks: [see details here](http://www.mattmahoney.net/dc/textdata)